test/integration-tests/test-utilities.js

```diff
@@ -19,9 +19,8 @@ async function exec(cmd) {
   console.debug(`\tRunning command: ${cmd}`);
   return new Promise((resolve, reject) => {
     shell.exec(cmd, { silent: false }, (err, stdout, stderr) => {
-      const error = err || stderr;
-      if (error) {
-        return reject(error);
+      if (err) {
+        return reject(stderr);
```

Comment:
Is this correct ? if `error` reject `stderr`?

Reply:
Yes, it is. If we return code of a bash command is not 0 then we reject a promise with error message (usually stderr)

Reply:
So maybe rename `err` to `errorCode`